src/main/resources/application.yaml

```diff
@@ -17,4 +17,13 @@ spring:
     default-property-inclusion: non_null
   liquibase:
     change-log: classpath:db.changelog/db.changelog-master.yaml
-    default-schema: tickets
\ No newline at end of file
+    default-schema: tickets
+  application:
+    name: TICKET-SERVICE
+
+eureka:
+  client:
+    register-with-eureka: true
+    fetch-registry: true
+    service-url:
+      defaultZone: http://localhost:8761/eureka/
```

Comment:
@eyakauleva  move host to env. don't hardcode it!

Reply:
fixed